tests/specs/spa/jsonp.e2e.js

```diff
@@ -191,7 +191,7 @@ describe('jsonp ajax events', () => {
       status: 200,
       type: 'ajax'
     }))
-    expect(event.callbackDuration).toBeGreaterThanOrEqual(2000)
+    expect(event.callbackDuration).toBeGreaterThanOrEqual(1999)
```

Comment:
This seems like an arbitrary value. What's our expectation with this test?

Reply:
<img width="397" alt="image" src="https://github.com/newrelic/newrelic-browser-agent/assets/28714891/b4a2cfed-0dea-4b20-949a-05521fa8b62d">
it's based on this; ios got `1999` here so I'm just giving it a slack of margin for small number diffs.

Reply:
Ah ok.